src/main/java/annotation/EmployeeProcessorTypeAnnotation.java

```diff
@@ -0,0 +1,14 @@
+package annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface EmployeeProcessorTypeAnnotation {
+
+    String createdBy() default "Andriy Kulyk";
+    String lastModified() default "5/7/2020";
```

Comment:
Using default field means that it's not mandatory to pass this paramether into Annotation. Let it be to the "created by",
but ir's not really logicaly for a "lastModified()", because date it's smth that will be diffent every time